src/cmdlib.sh

```diff
@@ -272,15 +276,28 @@ EOF
     if [ -n "${ref_is_temp}" ]; then
         echo 'ref: "'"${ref}"'"' >> "${override_manifest}"
     fi
+    local layers=""
     if [ -d "${configdir}/overlay" ]; then
         echo -n "Committing ${configdir}/overlay... "
         ostree commit --repo="${tmprepo}" --tree=dir="${configdir}/overlay" -b "${name}-config-overlay" \
           --owner-uid 0 --owner-gid 0 --no-xattrs --no-bindings --parent=none \
           --timestamp "${git_timestamp}"
-        cat >> "${override_manifest}" <<EOF
-ostree-layers:
-  - ${name}-config-overlay
-EOF
+        layers="${layers} ${name}-config-overlay"
+    fi
+    if [ -n "${parent_config}" ]; then
```

Comment:
This should also have `&& [ -d "${parent_config}/overlay" ]`, right?

Reply:
Yep, fixed thanks!